yamllint/conf/relaxed.yaml

```diff
@@ -6,9 +6,11 @@ rules:
   braces:
     level: warning
     max-spaces-inside: 1
+    max-spaces-inside-empty: -1
```

Comment:
Did you mean `1`?

If you really meant `-1`, you can remove these, because the `relaxed` rule extends `default`.